<launch>

  <!-- Declare argument to be passed in -->
  <arg name="n" />
  <arg name="control_law" />

  <node name="high_level_controller" pkg="dcsl_high_level_control" 
    type="dcsl_miabot_high_level_control.py" args="$(arg n) $(arg control_law)" >
  </node > 

</launch>
